lib/App/Notitia/Util.pm

```diff
@@ -969,6 +970,15 @@ sub to_msg (@) {
    my $k = shift; return $k, { no_quote_bind_values => TRUE, params => [ @_ ] };
 }
 
+
+sub contrast_colour {
+   my $background = shift;
+   my $po = new Graphics::ColorNames( qw( X ) );
```

Comment:
Indirect object syntax is strictly forbidden. Seriously that has to be Graphics::ColorNames->new( 'X' );